_layouts/default.html

```diff
@@ -1,9 +1,19 @@
 <!DOCTYPE html>
 <html>
-    <head>
-    </head>
-    <body>
-    {{ content }}
-        
-    </body>
+  <head>
+    <meta charset='utf-8' />
+    <meta name="description" content="{{site.name}}|{{page.title}}" />
+    <link rel="stylesheet" type="text/css" media="screen" href="{{ site.baseurl}}/stylesheets/stylesheet.css">
+    <title>{{site.name}}|{{page.title}}</title>
+  </head>
+  <body>
+{% include header.html %}
```

Comment:
indentation on this line seems wrong. compared to the footer one, this one has no whitespace prefix while the other one does

Reply:
if you mean the difference between Liquid and HTML is that they are 2 different things, at least I treat them differently

Reply:
I mean the first include is literally this:
```
{% include header.html %}
```
the 2nd one below is:
```
    {% include footer.html %}
```
2nd include statement has whitespace prefix, first one does not

Reply:
I see now, it was just a copy of the original page layout, so IDK what the author wanted to do in that case. OTOH I don't pay so much attention on HTML indentation (and so Liquid one if it's not my code) since it doesn't change the way the page works since it needs just tags to match.